lib/data/people_dao.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:sqflite/sqflite.dart';
+import 'package:sciencenotes/domain/users.dart';
+import 'package:sciencenotes/data/DBhelper.dart';
+
+
+class UserDao {
```

Comment:
Se for alterar o nome da tabela lá no banco, lembre de alterar os nomes aqui também.